### PR TITLE
Introduce workflow to update documentation

### PR DESCRIPTION
**Pull request by silamon**
Fixes https://github.com/xtermjs/xterm.js/issues/2606

When needed, the workflow can be triggered under Github Actions. It will update the docs and open a Pull Request.

### CODE REVIEW

File: .github/workflows/update-docs.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          cd _xterm.js && npm install mustache && npm run --silent vtfeatures > ../_docs/api/vtfeatures.md & cd ..
+      
+      - name: Fetch xterm.js version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
+        with:
+          path: _xterm.js
+      
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
```
Comment (Tyriar) [inline .github/workflows/update-docs.yml:48]: Can you pin these actions to commits instead of branches so I can review the code?

**Review by Tyriar — CHANGES_REQUESTED** [review]
Very cool if this works 🙂

File: .github/workflows/update-docs.yml (hunk 1 above)
Comment (silamon) [reply .github/workflows/update-docs.yml:48]: Still works: https://github.com/silamon/xtermjs.org/pull/4
I will pin them to the latest released tag with commit sha.

**Comment by silamon** [comment]
@Tyriar , updated everything today and it still works. If you don't want to merge this one, can you close it? I've also opened a PR to update the documentation to xterm 5.3.0 using this github action workflow that you can merge if you don't want to merge this one.

**Review by Tyriar — APPROVED** [review]
Thanks for the ping, totally forgot about this 😅 

Will merge as see how it goes 🤞

**Comment by Tyriar** [comment]
First run https://github.com/xtermjs/xtermjs.org/actions/runs/6207984904

**Comment by Tyriar** [comment]
Extremely cool! Thanks for fixing this pretty big problem 👏 https://github.com/xtermjs/xtermjs.org/pull/191

**Pull request merged**